gomi.go

```diff
@@ -0,0 +1,24 @@
+package sample_circleci
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
don't use an underscore in package name